osmosis-1/osmosis.zone_assets.json

```diff
@@ -1382,7 +1382,7 @@
     {
       "chain_name": "medasdigital",
       "base_denom": "umedas",
-      "path": "transfer/channel-519/umedas",
+      "path": "transfer/channel-87042/umedas",
```

Comment:
This updated IBC connection has to be registered

Reply:
Sorry saw it and forgot the pull request of the changed files, did it instantly..